### PR TITLE
Add changelog for 10.0.2, 9.1.6, 9.0.10

### PR DESCRIPTION
**Pull request by PVince81**
@phil-davis @jnweiger @eMerzh 

don't merge before these versions are out

### CODE REVIEW

File: page-changelog.php [hunk 1]
```diff
@@ -7,14 +7,59 @@
...
 <li><a href="#latest8.0">ownCloud 8.0 (unsupported!)</a></li>
 <li><a href="#latest7">ownCloud 7 (unsupported!)</a></li>
 <li><a href="#latest6">ownCloud 6 (unsupported!)</a></li>
 </ul>
 
 <a name="latest10.0"></a></li>
+<h3>Version 10.0.2 <small>May 30 2017</small></h3></li>
+Release notes: <a href="https://doc.owncloud.org/server/10.0/admin_manual/release_notes.html#changes-in-10-0-2">Release notes</a></br>
+<ul>
+<li>[major] Fix issue with database.xml migration being triggered twice on market app install: <a href="https://github.com/owncloud/core/issues/27982">core/#27982</a></li>
```
Comment (kdslkdsaldsal) [inline page-changelog.php]: Ending ``:`` where other entries are using `` - `` as a limiter between the text and the github issue

File: page-changelog.php [hunk 2]
```diff
@@ -7,14 +7,59 @@
...
+<li>[major] Apps formerly marked as shipped can now be uninstalled - <a href="https://github.com/owncloud/core/issues/27985">core/#27985</a></li>
+<li>[major] Market now properly updates app version when using multiple apps paths: <a href="https://github.com/owncloud/core/issues/27989">core/#27989</a></li>
```
Comment (kdslkdsaldsal) [inline page-changelog.php]: Ending ``:`` where other entries are using `` - `` as a limiter between the text and the github issue

File: page-changelog.php [hunk 3]
```diff
@@ -7,14 +7,59 @@
...
+MD5: <a href="https://download.owncloud.org/community/owncloud-9.0.10.tar.bz2.md5">owncloud-9.0.10.tar.bz2.md5</a> or <a href="https://download.owncloud.org/community/owncloud-9.0.10.zip.md5">owncloud-9.0.10.zip.md5</a></br>
+SHA256: <a href="https://download.owncloud.org/community/owncloud-9.0.10.tar.bz2.sha256">owncloud-9.0.10.tar.bz2.sha256</a> or <a href="https://download.owncloud.org/community/owncloud-9.0.10.zip.sha256">owncloud-9.0.10.zip.sha256</a></br>
+PGP (<a href="https://owncloud.org/owncloud.asc">Key</a>): <a href="https://download.owncloud.org/community/owncloud-9.0.10.tar.bz2.asc">owncloud-9.0.10.tar.bz2.asc</a> or <a href="https://download.owncloud.org/community/owncloud-9.0.10.zip.asc">owncloud-9.0.10.zip.asc</a></br>
+Packages on our Build Service: <a href="https://download.owncloud.org/download/repositories/9.0/owncloud">9.0 Release Channel</a> and <a href="https://download.owncloud.org/download/repositories/stable/owncloud/">Stable Release Channel</a>. Learn more about <a href="/release-channels">Release Channels</a>.
```
Comment (kdslkdsaldsal) [inline page-changelog.php]: 9.0.10 is not available in the stable release channel

**Comment by PVince81** [comment]
@kdslkdsaldsal thanks for spotting, fixed

File: page-changelog.php [hunk 4]
```diff
@@ -59,7 +59,6 @@
 MD5: <a href="https://download.owncloud.org/community/owncloud-9.0.10.tar.bz2.md5">owncloud-9.0.10.tar.bz2.md5</a> or <a href="https://download.owncloud.org/community/owncloud-9.0.10.zip.md5">owncloud-9.0.10.zip.md5</a></br>
 SHA256: <a href="https://download.owncloud.org/community/owncloud-9.0.10.tar.bz2.sha256">owncloud-9.0.10.tar.bz2.sha256</a> or <a href="https://download.owncloud.org/community/owncloud-9.0.10.zip.sha256">owncloud-9.0.10.zip.sha256</a></br>
 PGP (<a href="https://owncloud.org/owncloud.asc">Key</a>): <a href="https://download.owncloud.org/community/owncloud-9.0.10.tar.bz2.asc">owncloud-9.0.10.tar.bz2.asc</a> or <a href="https://download.owncloud.org/community/owncloud-9.0.10.zip.asc">owncloud-9.0.10.zip.asc</a></br>
-Packages on our Build Service: <a href="https://download.owncloud.org/download/repositories/9.0/owncloud">9.0 Release Channel</a> and <a href="https://download.owncloud.org/download/repositories/stable/owncloud/">Stable Release Channel</a>. Learn more about <a href="/release-channels">Release Channels</a>.
```
Comment (kdslkdsaldsal) [inline page-changelog.php]: That was one line killed to much 😄  Its available in the 9.0 release channel at:

https://download.owncloud.org/download/repositories/9.0/owncloud

but not at:

https://download.owncloud.org/download/repositories/stable/owncloud

so only the part of the stable release channel needs to be killed.

**Comment by PVince81** [comment]
@kdslkdsaldsal check it again 😄

File: page-changelog.php [hunk 5]
```diff
@@ -59,6 +59,7 @@
 MD5: <a href="https://download.owncloud.org/community/owncloud-9.0.10.tar.bz2.md5">owncloud-9.0.10.tar.bz2.md5</a> or <a href="https://download.owncloud.org/community/owncloud-9.0.10.zip.md5">owncloud-9.0.10.zip.md5</a></br>
 SHA256: <a href="https://download.owncloud.org/community/owncloud-9.0.10.tar.bz2.sha256">owncloud-9.0.10.tar.bz2.sha256</a> or <a href="https://download.owncloud.org/community/owncloud-9.0.10.zip.sha256">owncloud-9.0.10.zip.sha256</a></br>
 PGP (<a href="https://owncloud.org/owncloud.asc">Key</a>): <a href="https://download.owncloud.org/community/owncloud-9.0.10.tar.bz2.asc">owncloud-9.0.10.tar.bz2.asc</a> or <a href="https://download.owncloud.org/community/owncloud-9.0.10.zip.asc">owncloud-9.0.10.zip.asc</a></br>
+Packages on our Build Service: <a href="https://download.owncloud.org/download/repositories/9.0/owncloud">9.0 Release Channel</a>.
```
Comment (kdslkdsaldsal) [inline page-changelog.php]: Plus: ``Learn more about <a href="/release-channels">Release Channels</a>.`` ?

**Comment by PVince81** [comment]
ok added, how about now @kdslkdsaldsal ?

**Comment by kdslkdsaldsal** [comment]
👍 

The only thing what probably needs a PR at the documentation is:

https://doc.owncloud.org/server/10.0/admin_manual/release_notes.html#changes-in-10-0-2

which AFAICS doesn't exist.

**Pull request merged**